tools/mo/openvino/tools/mo/front/onnx/squeeze_ext.py

```diff
@@ -14,11 +14,14 @@ class SqueezeFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         axis = int64_array(onnx_attr(node, 'axes', 'ints', default=[]))
-
-        attrs = {
-            'squeeze_dims': axis if len(axis) != 0 else None
-        }
-
-        # update the attributes of the node
-        Squeeze.update_node_stat(node, attrs)
+        # check if axes attribute was provided, which is required for onnx opset < 13.
+        # Otherwise it means that onnx opset >= 13 and there are no attributes
+        if axis.any():
+            attrs = {
+                'squeeze_dims': axis if len(axis) != 0 else None
+            }
+            # update the attributes of the node
+            Squeeze.update_node_stat(node, attrs)
+        else:
+            Squeeze.update_node_stat(node)
```

Comment:
@rkazants It seems that the change is not necessary. The previous way of checking onnx_version with `get_onnx_opset_version(node)` fails in `squueze_ext_test.py` because in that test squeeze node is wrapped in helper class `PB` which does not have `graph.graph` object which is required to check onnx_version. So it seems that we either must change the test cases (do not use PB helper class) or remove checking onnx_version in ext file.